src/gps/gps.cpp

```diff
@@ -172,10 +177,29 @@ parse_term(gps_t* gh) {
 #if GPS_CFG_STATEMENT_GPGSA
         } else if (!strncmp(gh->p.term_str, "$GPGSA", 6) || !strncmp(gh->p.term_str, "$GNGSA", 6)) {
             gh->p.stat = STAT_GSA;
+            /* This message is a delimiter to indicate all GSV messages have been received.
+               Reset the counter of satellites in view for the next cycle. */
+            gSivTotal = 0;
 #endif /* GPS_CFG_STATEMENT_GPGSA */
 #if GPS_CFG_STATEMENT_GPGSV
-        } else if (!strncmp(gh->p.term_str, "$GPGSV", 6) || !strncmp(gh->p.term_str, "$GNGSV", 6)) {
+        } else if( !strncmp(gh->p.term_str, "$GPGSV", 6) ) {
```

Comment:
There are a lot of strncmp calls.  Ideally we'd want to pattern match to `$G([PNLABQ])GSV`.  Consider this instead:
```cpp
} else if( !strncmp(gh->p.term_str, "$G", 2) && !strncmp(&gh->p.term_str[3], "GSV", 3) ) {
    switch (strncmp(&gh->p.term_str[2])
```

Reply:
Thanks for suggestion. Will implement.